.github/workflows/homebrew-update.yaml

```
@@ -0,0 +1,55 @@
+name: "Update SuperDB Homebrew Cask"
+
+on:
+  schedule:
+    - cron: '5 8 * * *'
+  workflow_dispatch:
+
+jobs:
+  build-and-update:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout homebrew-tap
+        uses: actions/checkout@v5
+        with:
+          token: ${{ secrets.PAT_TOKEN }}
+      - name: Checkout super
+        uses: actions/checkout@v5
+        with:
+          repository: brimdata/super
+          path: super
+      - name: Check if super main has changed
+        id: changed
+        run: |
+          ( [ ! -f Casks/super.rb ] || [ $(awk '/version/ {print $2}' Casks/super.rb | head -1 | tr -d '"') != $(git -C super rev-parse --short=7 HEAD) ] ) && echo "changed=true" >> $GITHUB_OUTPUT || true
```

Nit: Commit the first Cask/super.rb yourself so you can assume it exists.

```suggestion
          [ $(awk '/version/ {print $2}' Casks/super.rb | head -1 | tr -d '"') != $(git -C super rev-parse --short=7 HEAD) ] && echo "changed=true" >> $GITHUB_OUTPUT || true
```